.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 executors:
   cicd82:
     docker:
-      - image: wunderio/silta-cicd:circleci-php8.2-node18-composer2-v1
+      - image: wunderio/silta-cicd:circleci-php8.2-node20-composer2-v1.2
```

Comment:
Isn't it so that we should avoid using `v1.x` tag and just use `v1` unless there's a need for a specific point release? 

With the v1.2, my understanding is that it would always use image v1.2.x but not receive updates for v1.3, v1.4 etc ?